lib/WTSI/NPG/HTS/PacBio/Sequel/RunPublisher.pm

```diff
@@ -47,6 +47,9 @@ Readonly::Scalar my $CCS_REPORT_COUNT       => 6;
 Readonly::Scalar my $DATA_LEVEL_TWO         =>
   $WTSI::NPG::HTS::PacBio::Sequel::AnalysisPublisher::DATA_LEVEL;
 
+## no critic (ValuesAndExpressions::ProhibitLeadingZeros)
+Readonly::Scalar my $MODE_GROUP_WRITABLE => 0020;
+## use critic
```

Comment:
```suggestion
Readonly::Scalar my $MODE_GROUP_WRITABLE => q(0020);
```
Better than the `no critic`? Pretty sure this is still interpreted as octal in `_dir_group_writable` in the bitwise `&`